src/everything/everything.ts

```diff
@@ -621,7 +617,7 @@ export const createServer = () => {
       );
       return {
         content: [
-          { type: "text", text: `LLM sampling result: ${result.content.text}` },
+          { type: "text", text: `LLM sampling result: ${Array.isArray(result.content) ? result.content.map(c => c.type === "text" ? c.text : JSON.stringify(c)).join("") : (result.content.type === "text" ? result.content.text : JSON.stringify(result.content))}` },
```

Comment:
This is necessary because of breaking changes in the MCP specification around sampling, see https://github.com/modelcontextprotocol/modelcontextprotocol/issues/1577